app/src/main/java/com/example/pygmyhippo/organizer/EventDB.java

```diff
@@ -88,21 +92,87 @@ public void getEvent(String eventID, DBOnCompleteListener<Event> listener) {
                 });
     }
 
+    /**
+     * Gets a long-live download uris for Picasso to get image from Firebase Storage.
+     * @author James
+     * @param url String rul with gs:// link to Firebase Storage image.
+     * @param listener DBOnCompleteListener to call when query completes.
+     */
+    public void getImageDownloadUrl(String url, StorageOnCompleteListener<Uri> listener) {
```

Comment:
could this make more sense in the ImageStorage class in the database folder?

Reply:
😭😭😭😭😭😭😭😭Yah it could :(